frontend/src/lib-components/molecules/modals/SessionExpiredModal.tsx

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2017-2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React from 'react'
+import styled from 'styled-components'
+
+import { Button } from 'lib-components/atoms/buttons/Button'
+
+import { useTranslations } from '../../i18n'
+import { defaultMargins } from '../../white-space'
+
+import BaseModal from './BaseModal'
+
+interface Props {
+  isOpen: boolean
+  onClose: () => void
+}
+
+const SessionExpiredModal: React.FC<Props> = ({ isOpen, onClose }) => {
+  const i18n = useTranslations()
+
+  return (
+    isOpen && (
+      <BaseModal
```

Comment:
The paddings etc seem a bit off. Did you try using InfoModal? I think you can pass everything needed to it and not e.g. render the buttons yourself.

Reply:
En alkanut hienosäätämään tätä vielä kohdilleen, kun tämä dialogi näkyy niin pienelle joukolle käyttäjiä. Korjataan tämä kohdilleen sitten jos joskus toteutetaan koko sovelluksen laajuinen uudelleenkirjautuminen session aikakatketessa.